src/civic/index.js

```diff
@@ -630,13 +646,15 @@ const upload = async ({
         }
     }
 
+    // Main loop on recordsById
     for (const [sourceId, recordList] of Object.entries(recordsById)) {
         if (previouslyEntered.has(sourceId) && !ignoreCache) {
             counts.exists++;
             continue;
         }
         if (purgeableEvidenceItems.has(sourceId)) {
-            // this should never happen, but if it does we have made an invalida assumption about how civic uses IDs
+            // this should never happen.
+            // If it does we have made an invalid assumption about how civic uses IDs.
```

Comment:
minor: generally I try to avoid multiple inline comments in a row. For this one you don't need multiple lines b/c comments, urls, and strings should be allowed to violate the line length limit. we can just wrap in the editor

Reply:
ok good